### ISSUE TITLE
logback.xml overrides client project's logger settings

### ISSUE THREAD

**Issue by ldoblies** [binance/binance-futures-connector-java#12]
Is it possible to remove logback.xml from the project? The file from the jar overrides client's logging settings and can even lead to warnings (multiple logback.xml files) if clients also use logback, which is not desirable. With gradle it seems to be hard to exclude this file.

**Comment by aisling-2** [comment]
Definitely. We already have an internal ticket to solve this in a future major version upgrade.

**Comment by taran2laa** [comment]
Any updates for this one? version 3.0.4 still has this problem.
